temel_bilgiler/giriş/web-geliştirmeye-giriş.md

```diff
@@ -0,0 +1,97 @@
+
+Bir web danışmanlık şirketi için çalışmak, freelance ücret potansiyelinizin bir kısmından vazgeçmenizi ama kod üzerinde daha fazla odaklanmanızı sağlar. Bu seçenek aynı zamanda iyi bir iş/yaşam dengesi ve ücret sunar.
+
+Son olarak, büyük, eski şirketler hala web geliştiricilere ihtiyaç duyar. Bu şirketler iyi bir iş/yaşam dengesi, ücret ve yan haklar sunar, ancak teknolojiye çok odaklanan bir şirketten daha yavaş aksiyon alabilirler.
+
+### Araçlar
+
+Aşağıda yaygın olarak kullanılan bazı araçlar var. Şu anda ne olduklarını bilmeyebilirsiniz, ancak ileride kesinlikle bileceksiniz.
+
+- **Bilgisayar**
+- **[Google](https://www.google.com/)**
+- **Text Editor**
+- **Command Line Interface (CLI)**
+- **[Stack Overflow](http://stackoverflow.com/)**
+- **[Git](https://git-scm.com/)**
+- **[GitHub](https://github.com/)**
+
+### Motivation
+
+Kod yazmayı öğrenmek inanılmaz derecede ödüllendirici olduğu kadar zor ve sinir bozucu da olabilir. Bir öğrenci olarak sahip olabileceğiniz en güçlü özellik, bir şeyler inşa etme isteği, problem çözme yeteneği ve engellerle karşılaştığınızda ısrarla üstüne gitmektir.
+
+Web geliştirme endüstrisi, farklı geçmişlere sahip başarılı geliştiricilerin uzun bir geçmişine sahiptir, bu nedenle insanlar genellikle oraya nasıl geldiğinizden daha çok ne yaptığınızı önemser.
+
+### Why Odin?
+
+Şunu bilmenizi isterim ki, bu kolay olmayacak.
+
+Birçok online öğrenme içeriği var, ancak çoğu, çok izole ve kontrol edilen bir ortamda öğretiliyor ve sadece belirli bir konuyu kapsıyor.
+
+Odin Projesi, neye ihtiyacınız olduğunu ve bir işe başlamak için neye ihtiyacınız olduğunu gerçekçi bir şekilde ele alır ve bir işe başladığınızda yapacağınız şeylere benzer bir ortamda çalışmanızı sağlar.
+
+Sektöre girebilmeniz için geniş bir beceri yelpazesine ve dillere ihtiyacınız olduğunu kabul eder.
+
+Odin Projesi, web geliştirme endüstrisindeki en iyi uygulamaları öğretir ve size gerçek dünyadaki geliştiricilerin kullandığı araçları kullanma fırsatı verir.
+
+Kendinizi bir araçta rahat hisseetmeye başladığınızda bir açık kaynaklı projeye katkıda bulunmaya başlayın [(Odin Projesinin kendisi gibi](/contributing). Ne yapabileceğiniz hakkında ne kadar çok şey öğrenirseniz, işe alınmanıza o kadar yaklaşırsınız. Bu projeler ayrıca özgeçmişinizde harika görünecek!
+
+### Conclusion
+
+Imarım bir web geliştiricisinin aslında ne yaptığını ve bunu bir kariyer olarak seçerseniz hayatınızın nasıl olacağını daha iyi anlamışsınızdır. Bu, web geliştirme dünyasına bir ön gösterim oldu.
+
+Bu "Temeller" kursunda, sonunda bilmeniz gereken konuların tüm yelpazesinde bir yolculuğa çıkacaksınız. Bu kurs, tamamen yabancı olabileceğiniz birçok konuya atlayarak, her birine küçük örnekler vererek ve sonra derinleşerek ilerliyor.
+
+Bu kursu tamamladıktan sonra, aşağıdaki kurslar bu konulara derinlemesine dalacak. İşe alınmanızı sağlayacak becerileri sağlamlaştırmak için onlarca satır kod yazacak, projeler gelitştirecek ve web sitesi oluşturacaksınız.
```

Comment:
fix: Sonraki kurslar da bu konulara derinlemesine dalacak. İşe alınmanızı sağlayacak becerileri sağlamlaştırmak için onlarca satır kod yazacak, proje geliştirecek ve web sitesi oluşturacaksınız.